server/db/mongodb/adapter.go

```diff
@@ -40,7 +43,8 @@ const (
 	adpVersion  = 111
 	adapterName = "mongodb"
 
-	defaultMaxResults = 1024
+	// This is capped by the Session's send queue limit (128).
+	defaultMaxResults = 100
```

Comment:
I think it should be left at 1024, while `defaultMaxMessageResults` should be defined as 100.

Reply:
BTW, why not keep it at a slightly higher value, like 112 or 120?

Reply:
Ah, sure. Thanks for catching. Apparently, my compile command does mysql adapter only.